### PR TITLE
[PLATFORM-730] Longer ids for the rescue!(?)

### PR DESCRIPTION
**Pull request by mondoreale**
**tl;dr** Chrome exposes elements with ids as `window` properties, i.e. `<div id=“a”></div>` is accessible via `window.a`. Longer ids are less likely to cause conflicts with… browser extensions.

I don't feel like it's a proper solution to the problem. Why? The exception tells us that `indexOf` is not a function. It means that `a[b].target.className` is somehow defined. Here's what Chrome thinks `window.a` is on core pages:

Screenshot 2019-05-28 18 06 05

_(`window.b` is part of the same svg, so `a[b]` kinda makes sense.)_

I'd treat this PR as an attempt to solve the issue rather than the actual fix. Hope that's good enough.

FYI, [an article](https://medium.com/@amir.harel/a-b-target-classname-indexof-is-not-a-function-at-least-not-mine-8e52f7be64ca) what helped me identify the issue

### CODE REVIEW

**Comment by mondoreale** [comment]
Adding more of you as reviewers cause it's an interesting case. Hope you're ok with that.

**Comment by timoxley** [comment]
One terrible alternative solution could be:

```
SVGAnimatedString.prototype.indexOf = Function.prototype
```

**Comment by tumppi** [comment]
Really interesting indeed. Maybe it's least bad to go with what Tim suggested. Although I'd change it to actually implement `indexOf`.

**Comment by mondoreale** [comment]
I'd vote against our `indexOf`. Usually it returns a number. `Function.prototype` doesn't. What if we go with `() => -1` or `() => 0`? Then we're blindly choosing an unknown path in someone else's code, meaning, what if `….indexOf() === -1` break the entire Internet and we helped? 😬 

@timoxley I couldn't reproduce the exact exception. I did manage to get a different/similar one. One that indicates that it's the global `a`. Here's how I did it:

May-29-2019 14-15-31

_(The thing I clicked in the video is Google Translate extension for Chrome, recently installed.)_

It's a little disappointing that my fix (changed ids) replaces the `not a function` exception with something like this:

Screenshot 2019-05-29 14 17 24

You just can't help these people! XD

I'd vote for unique ids (something we can control). Bugs in extensions we cannot control. We may as well just ignore the bug in Sentry. Not our zoo, not our monkeys.

FYI I may respond slower today. I'm afk.

Thanks guys!

**Comment by fonty1** [comment]
Able to replicate by translating those pages into German with the google translate extension. Wasn't able to confirm it fixed the issue because the PR build has some weird unrelated CORS errors.

Interesting problem, though probably doesn't break anything. I vote for the unique IDs, easy and benign change.

**Comment by mondoreale** [comment]
@fonty1 @timoxley @tumppi I went through our svgs and the `SvgIcon` component and removed nearly all ids. Re-exported `profileMan`, etc. The only svgs that need to have ids are the ones we consciously generate and I made them a little harder to conflict with.

Does it make sense now?
